packages/strapi-database/lib/queries/paginated-queries.js

```diff
@@ -35,8 +35,8 @@ const withDefaultPagination = params => {
   const { page = 1, pageSize = 100, ...rest } = params;
 
   return {
-    page: parseInt(page),
-    pageSize: parseInt(pageSize),
+    page: Math.max(1, parseInt(page)) || 1,
```

Comment:
On the DB layer it would make more sense to throw error if the page or pageSize are invalid instead. 

This kind of logic can be done but in the controllers / services instead